gdcdictionary/schemas/sample.yaml

```diff
@@ -551,7 +550,6 @@ properties:
       - Xenograft Tissue
       - Unknown
       - Not Reported
-      - Not Allowed To Collect
   sample_type_id:
```

Comment:
Do we want to remove `sample_type_id` - not sure it makes much sense in the context of KF?

Reply:
Yes. I fully support removing it.

Reply:
Removed `sample_type_id`.